runelite-client/src/main/java/net/runelite/client/plugins/banktags/TagManager.java

```diff
@@ -109,4 +129,49 @@ public void removeTag(int itemId, String tag)
 			setTags(itemId, tags);
 		}
 	}
+
+	private boolean testClue(int itemId)
+	{
+		ClueScroll c = clueScrollService.getClue();
+
+		if (c != null)
```

Comment:
flip this if (if c == null) return false

Reply:
that would look ugly though, having a return false in front and at the end

Reply:
nah, imo it would make it a bit easier to read, and reduce nesting what is always good thing